report/sections/Chapter 11 - Reflection/Project-management.tex

```diff
@@ -0,0 +1,45 @@
+
+\begin{itemize}
+    \item Not started.
+    \item In progress.
+    \item Review required.
+    \item Done.
+\end{itemize}
+
+While Notion provides fewer features in terms of integration with Github, the tool was familiar to us and provided the functionality that we needed to manage the project.
+Once we switched to Notion, planning and updating the tasks became a more active part of our daily work. This led to better productivity as we were now able to better maintain an overview of what each member of the group was doing and and how far they were with each task.
+
+\section{Agile development approach} \label{sec:agile-dev}
+Irrespective of which tools we used, the goal from the beginning of the project was to use a SCRUM-like approach to project management. 
+Generally, SCRUM is a framework that helps teams and organizations generate value through flexible solutions to complex problems. SCRUM is an iterative and incremental approach to project work where the project is broken up into sprints. A sprint is timeframe, usually 1-4 weeks in length, in which the the teams tries to complete the work that has been selected from a product backlog.
+
+In the spirit of the SCRUM mindset we only adopted the specific things that we felt were necessary. For our project we did not have a SCRUM master or product owner. Instead we collectively created tasks based on features that we wanted to implement. 
+Additionally, instead of traditional sprints we decided on a minimally viable product, a product that would satisfy the project goals without any extra features, and divided that into sub parts. We would then generate tasks that would amount to the completion of that subtask and set a date for the expected completion. While this approach in essence is similar to a sprint, we did not think of it as a sprint. 
+We also did not assign any points to the tasks in order to estimate the time needed to complete tasks. Instead these were just taken as a dicussion each morning, where we would brief each other on the estimated time remaining to complete the task.
+Finally, we also approached review and retrospect differently in that we would handle potential process changes as issues arose. An example of this being the decision to switch to Notion. Similarly, discussions about internal team issues were taken regularly.
+
+This approach was based on previous experience working with a more rigid SCRUM methodology in a school project setting which we had found to be less productive. We felt that a more fluid approach to SCRUM worked the best especially due to the limited amount of work hours available dedicated specifically to the project per week.
```

Comment:
Fine catch-all explanation, but maybe put it earlier, so the reader doesn't think what I did (see previous comments about explaining decisions).